keras_contrib/objectives.py

```diff
@@ -1,10 +1,62 @@
 from __future__ import absolute_import
 
-from . import backend as K
+import keras_contrib.backend as KC
 from keras.utils.generic_utils import get_from_module
 
 from keras.objectives import *
 
 
 def get(identifier):
     return get_from_module(identifier, globals(), 'objective')
+
+
+class DSSIMObjective():
+    def __init__(self, batch_size, k1=0.01, k2=0.03, kernel_size=3, max_value=1.0):
+        """
+        Difference of Structural Similarity (DSSIM loss function). Clipped between 0 and 0.5
+        Note : You shoul add a regularization term like a l2 loss in addition to this one.
+        :param batch_size: Batch size used in the model
+        :param k1: Parameter of the SSIM (default 0.01)
+        :param k2: Parameter of the SSIM (default 0.03)
+        :param kernel_size: Size of the sliding window (default 3)
+        :param max_value: Max value of the output (default 1.0)
+        """
+        self.__name__ = "DSSIMObjective"
+        self.batch_size = batch_size
```

Comment:
Does this handle the case where the final batch in an epoch is < batch_size (e.g. num_samples % batch_size != 0)?